package.json

```diff
@@ -104,6 +105,7 @@
     "react-onclickoutside": "^6.6.2",
     "react-password-strength": "^2.4.0",
     "react-redux": "^5.0.6",
+    "react-router-dom": "^4.3.1",
```

Comment:
this should be added to webpack.vendor.js as its used by Katello as well (same for history if its really required).